src/providers/UserStateProvider.tsx

```diff
@@ -65,9 +61,7 @@ export default function UserStateProvider({
         ? getGaugeRewardsUserData(
             library,
             chainId,
-            (Object.values(gauges.gauges) as Gauge[]).map(
-              ({ address }) => address,
-            ),
+            Object.values(gauges.gauges).map(({ address }) => address),
```

Comment:
was that type unneeded?

Reply:
The lint message was that typing this is redundant.